.mergify.yml

```diff
@@ -18,8 +18,35 @@ pull_request_rules:
         strict_method: merge
       delete_head_branch: {}
     conditions:
+      - base!=release
+      - -title~=(WIP|wip)
+      - -label~=(blocked|do-not-merge|no-squash)
+      - -merged
+      - -closed
+      - author!=dependabot[bot]
+      - author!=dependabot-preview[bot]
+      - "#approved-reviews-by>=1"
+      - -approved-reviews-by~=author
+      - "#changes-requested-reviews-by=0"
+      - status-success~=AWS CodeBuild us-east-1
+      - status-success=Semantic Pull Request
```

Comment:
Weird that we have to explicitly state these. Can't we just say, all 'mandatory checks' (as configured in the repo's settings) must pass? 

Is this what the next line says? in which case, can we drop these?

Reply:
I don't know. I don't want to fiddle more with this than what I'm trying to achieve. I'm assuming the rest of the rules are there for a reason.